src/package/midlrt/Microsoft.Windows.MidlRT.targets

```diff
@@ -0,0 +1,339 @@
+﻿<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <PropertyGroup>
+        <!-- Only do this for MSBuild versions below 16.0
+             as it is since done automatically, see https://github.com/microsoft/msbuild/pull/3605-->
+        <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)'  &lt;= '15'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <MidlRTProjectWinMD>$(OutDir)$(RootNamespace).winmd</MidlRTProjectWinMD>
+        <MidlRTMergedDir>$(IntDir)Merged\</MidlRTMergedDir>
+        <MidlRTUnmergedDir>$(IntDir)Unmerged\</MidlRTUnmergedDir>
+
+        <ModernMidlRT Condition="'$(ModernMidlRT)'==''">true</ModernMidlRT>
+
```

Comment:
to be safe, you should normalize:
        <ModernMidlRT Condition="'$(ModernMidlRT)' != 'true'">false</ModernMidlRT>

otherwise comparisons with 'false' below might fail